doc/check.texi

```diff
@@ -1217,6 +1217,17 @@ would pass.
 @item ck_assert_ptr_eq
 @itemx ck_assert_ptr_ne
 
+Compares a pointers against null and displays predefined message with
+condition and value of the input parameter on failure.
+@code{@item ck_assert_ptr_null} checks that pointer is equal to NULL and
```

Comment:
Sorry, I did it again. I would like to know how do I make such mistakes. Moreover, I've mixed up descriptions.

If checks in my commits fail, you can just wait until I fix everything. I make changes in a hurry, but I can fix them all only when I get home. So I see what's wrong but just can't send fixes.